tiatoolbox/models/engine/patch_predictor.py

```diff
@@ -452,6 +452,289 @@ def _predict_engine(
 
         return cum_output
 
+    def _predict_patch(self, imgs, labels, return_probabilities, return_labels, on_gpu):
```

Comment:
Move to be above the `_predict_tile_wsi`

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/365/commits/fb2c52ef882a0e90fe0427d00c617073852b01ff